programs/graph_program/src/instructions/make_connection.rs

```diff
@@ -21,6 +26,8 @@ pub fn make_connection_instruction(ctx: Context<MakeConnection>, to: Pubkey) ->
     let connection = &mut ctx.accounts.connection;
     connection.from = ctx.accounts.from.key();
     connection.to = to;
+    connection.connected_at = Some(ctx.accounts.clock.unix_timestamp);
+    connection.status = Some(ConnectionStatus::Connected);
```

Comment:
```suggestion
    connection.connected_at = Some(ctx.accounts.clock.unix_timestamp);
    connection.disconnected_at = None;
```
I think the timestamps connected_at and disconnected_at are all that are needed. status is a spot we can save some bytes.

Since connections are derived from the from and to every make connection should set the disconnected_at to none.

Reply:
Yep, just removed this!